framework/src/main/java/org/checkerframework/common/reflection/DefaultReflectionResolver.java

```diff
@@ -487,47 +488,51 @@ private boolean isUnknownMethod(MethodInvocationTree tree) {
         Names names = Names.instance(context);
 
         List<Symbol> result = new ArrayList<>();
+        Method loadClass = null;
```

Comment:
Don't initialize local variables to dummy `null` values. Like this, the definite assignment checks by the compiler will not warn you if you missed a branch.